### ISSUE TITLE
deleteAllAlarms

### ISSUE THREAD

**Issue by matheusf31** [liplylie/react-native-simple-alarm#5]
Hi there, i got this error when i try to run deleteAllAlarms() in android emulator:

```
[Sat Jul 18 2020 12:13:43.453]  LOG      [TypeError: storage.forEach is not a function. (In 'storage.forEach(function (_ref) {
              var oid = _ref.oid;
              cancelAlarmById(oid);
            })', 'storage.forEach' is undefined)]
```

**Comment by liplylie** [comment]
@matheusf31 I'll take a look into this

**Comment by liplylie** [comment]
Fixed in [1.0.4]

[1.0.4]: https://github.com/liplylie/react-native-simple-alarm/releases/tag/1.0.4
